tests/ui/rfcs/rfc-2632-const-trait-impl/effects/no_explicit_const_params.rs

```diff
@@ -0,0 +1,27 @@
+#![feature(const_trait_impl, effects)]
```

Comment:
Can you add an aux-build test to see what happens when you try to specify the turbofished host param of a *foreign* function (with effects enabled) in a local crate (with effects disabled)?